airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -196,7 +196,7 @@ private void reportSuccess(final ConnectionUpdaterInput connectionUpdaterInput)
   private void reportFailure(final ConnectionUpdaterInput connectionUpdaterInput) {
     jobCreationAndStatusUpdateActivity.attemptFailure(new AttemptFailureInput(
         connectionUpdaterInput.getJobId(),
-        connectionUpdaterInput.getAttemptNumber()));
+        connectionUpdaterInput.getAttemptId()));
```

Comment:
We still need to use the ID here, how ever we need to use the maybeAttemptId.get (like in the report success method)